server/index.ts

```diff
@@ -4,12 +4,17 @@ import morgan from 'morgan';
 import nextjs from 'server/lib/next';
 import { responseErrorHandler } from 'express-response-errors';
 import exampleRouter from 'server/routers/exampleRouter';
+import chapterRouter from 'server/routers/chapter';
+
+import { initSequelize } from 'server/models';
 
 const app: express.Application = express();
 
 nextjs.nextApp.prepare().then(async () => {
   const port = process.env.PORT || 8000;
 
+  initSequelize();
```

Comment:
is this enough or do we have do something like `await initSequelize().authenticate()`?  connecting to the db should be async